rosdep/python.yaml

```diff
@@ -557,6 +557,13 @@ python-astor-pip:
   ubuntu:
     pip:
       packages: [astor]
+python-astropy-pip:
```

Comment:
the package only seems to be available in python3, but this is a python2 entry.
Please use `python3-astropy-pip` for python3 packages.

Reply:
Good point. Thank you. Changed